python/pyspark/sql/connect/column.py

```diff
@@ -107,12 +106,16 @@ def _to_expr(v: Any) -> Expression:
     return v._expr if isinstance(v, Column) else LiteralExpression._from_value(v)
 
 
-@with_origin_to_class(["to_plan"])
 class Column(ParentColumn):
     def __new__(
         cls,
         expr: "Expression",
     ) -> "Column":
+        from pyspark.errors.utils import with_origin_to_class
```

Comment:
can we add some comments? This is pretty hacky ..

Reply:
Sure! Just added a comment